src/styles/blocks/catalog.scss

```
@@ -17,9 +17,17 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-property: transform, color;
+    transition-duration: $smoothly-during, $smoothly-during;
```

Also you could combine it
```suggestion
    transition: transform $smoothly-during, color $smoothly-during;
```